package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "opentelemetry",
```

Comment:
Should this be `opentelemetry-base` similar to the `@opencensus/opencensus-base` for OpenCensus?

Reply:
Done. Thanks